this doesn't install an actual library with headers and an soname, so there is
no point to install it at all
--
diff --git a/libhexagonrpc/meson.build b/libhexagonrpc/meson.build
index d83a9a4..a913cf1 100644
--- a/libhexagonrpc/meson.build
+++ b/libhexagonrpc/meson.build
@@ -1,10 +1,9 @@
-libhexagonrpc = shared_library('hexagonrpc',
+libhexagonrpc = static_library('hexagonrpc',
   'context.c',
   'fastrpc.c',
   'interfaces.c',
   'session.c',
   c_args : cflags,
   include_directories : include,
-  install : true
 )

